decentraland/unity-explorer
# Fix: camera reel screencapture hud hide UI issues

## What does this PR change?

- locked shortcuts when in the screencapture mode
- fixed hide/show ui issues 
- refactored screencapture shortcuts to not have specific Controller for it

## How to test the changes?

1. Launch the explorer
2. Verify shortcuts for emotes and other features are not working when in screencapture mode
3. Verify show/hide UI doesn't make any issues
4. Verify show/hide screencapture HUD doesn't make issues

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
